Spanner/src/Database.php

```diff
@@ -1146,19 +1171,70 @@ public function delete($table, KeySet $keySet, array $options = [])
      *
      * ```
      * // Array parameters which may be null or empty must include the array value type.
+     * use Google\Cloud\Spanner\ArrayType;
+     *
      * $result = $database->execute('SELECT @emptyArrayOfIntegers as numbers', [
      *     'parameters' => [
      *         'emptyArrayOfIntegers' => []
      *     ],
      *     'types' => [
-     *         'emptyArrayOfIntegers' => [Database::TYPE_ARRAY, Database::TYPE_INT64]
+     *         'emptyArrayOfIntegers' => new ArrayType(Database::TYPE_INT64)
```

Comment:
Is this a breaking change?

Reply:
No. The old way still works, but to support structs nested within arrays I needed to add this class, and to limit confusion the documentation suggests only the new way. [This unit test](https://github.com/GoogleCloudPlatform/google-cloud-php/pull/1033/files#diff-acea5a7e9c7cbe20643bd49fe42d93b8R164) demonstrates the equivalence of the two methods of defining the type of an array. (see the dataProvider method immediately after the test case).

Reply:
Ok. Also currently is there a way for user to look at the type of column in a row? How is the type of a column which is `array<struct>` returned to the user?

Reply:
Yep! [`Google\Cloud\Spanner\Result::metadata()`](https://github.com/GoogleCloudPlatform/google-cloud-php/blob/master/Spanner/src/Result.php#L269) will return the `ResultSetMetadata` response.